portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -80,11 +84,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String text = (String) entity.getProperty("text");
       long timestamp = (long) entity.getProperty("timestamp");
 
+      // Translate text
+      text = translate(request, text);
```

Comment:
Since translation doesn't work when run locally, to avoid comments completely failing, maybe you can do a `try-catch` here, and just leave the text untranslated if an exception occurred during translation.